services/pdf-ms/lambdas/addPdfToCase.ts

```diff
@@ -44,6 +44,29 @@ export async function main(event: Record<string, any>): Promise<Boolean> {
   const ssmParams = await PDF_SSM_PARAMS;
   const submittedRecurringAnswers = submittedCase.forms[ssmParams.recurringFormId].answers;
 
+  /**
+   * >>>WARNING Ugly shit ahead! WARNING<<<
+   *
+   * Injects the Viva application period date onto the PDF using template(PDF) variable {{periodDate}}
+   *
+   */
+  const { period } = submittedCase.details;
+  const dateOptions = { timeZone: 'Europe/Stockholm' };
+  const startDatePeriod = new Date(period.startDate).toLocaleDateString('sv-SE', dateOptions);
+  const endDatePeriod = new Date(period.endDate).toLocaleDateString('sv-SE', dateOptions);
```

Comment:
This is my personal opinon and does not have any implications on the code functionalty.
I rather have the name to read out periodStartDate, periodEndDate. Since it follows other variabels better ie period.startDate, period.endDate and periodDate.

Reply:
Ye. the thing is, it's somewhat harder to distinguish the two by doing it this way

periodStartDate
periodEndDate

vs

startDatePeriod
endDatePeriod

maybe this is better still?

const pdfPeriodDate = {
  start: new Date(period.startDate).toLocaleDateString('sv-SE', dateOptions),
  end: new Date(period.endDate).toLocaleDateString('sv-SE', dateOptions),
};

submittedRecurringAnswers.push({
  field: {
    id: 'periodDate',
    tags: [],
  },
  value: `${pdfPeriodDate.start} – ${pdfPeriodDate.end}`,
});